package.json

```diff
@@ -32,6 +32,7 @@
   },
   "devDependencies": {
     "@types/eslint": "^8.4.1",
+    "@types/multer": "^1.4.7",
```

Comment:
#64 już to dodał do `packages/server/package.json` więc w tym w roocie nie ma co tego duplikować.

Reply:
tak, zostało ze sprawdzania jak to zrobić w swaggerze żeby załadować plik :D